plugins/woocommerce-admin/client/customize-store/assembler-hub/sidebar/sidebar-navigation-screen-homepage-ptk.tsx

```diff
@@ -35,7 +35,6 @@ import { ADMIN_URL } from '~/utils/admin-settings';
 import { SidebarNavigationScreen } from './sidebar-navigation-screen';
 
 import { trackEvent } from '~/customize-store/tracking';
-import { FlowType } from '~/customize-store/types';
```

Comment:
Nice catch! ESlint should scream on us on unused import right? Maybe we should work on it in cooldown?

Reply:
I noticed this because the lint CI job failed!